src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -20,19 +20,7 @@
 
 #include "openvino/pass/low_latency.hpp"
 #include "openvino/pass/manager.hpp"
-
-using Version = ov::pass::Serialize::Version;
-
-inline Version convert_to_version(const std::string& version) {
-    if (version == "UNSPECIFIED")
-        return Version::UNSPECIFIED;
-    if (version == "IR_V10")
-        return Version::IR_V10;
-    if (version == "IR_V11")
-        return Version::IR_V11;
-    throw ov::Exception("Invoked with wrong version argument: '" + version +
-                        "'! The supported versions are: 'UNSPECIFIED'(default), 'IR_V10', 'IR_V11'.");
-}
+#include "pyopenvino/core/common.hpp"
```

Comment:
do it need to be included?
As I see `convert_to_version` was used only in `serialize` method

Reply:
removed